pkg/client/redhatsso/client.go

```diff
@@ -67,6 +72,80 @@ type rhSSOClient struct {
 	cache         *cache.Cache
 }
 
+// AcsClientResponseData ...
+type AcsClientResponseData struct {
+	Name         string `json:"name,omitempty"`
+	ClientID     string `json:"clientId,omitempty"`
+	ClientSecret string `json:"secret,omitempty"`
+}
+
+type acsClientRequestData struct {
+	Name         string   `json:"name,omitempty"`
+	OrgID        string   `json:"orgId,omitempty"`
+	RedirectUris []string `json:"redirectUris,omitempty"`
+}
+
+func (c *rhSSOClient) CreateAcsClient(accessToken string, name string, orgID string, redirectUris []string) (*AcsClientResponseData, error) {
+	client := &http.Client{}
+	requestData := &acsClientRequestData{
+		Name:         name,
+		OrgID:        orgID,
+		RedirectUris: redirectUris,
+	}
+	bodyBytes, err := json.Marshal(requestData)
+	if err != nil {
+		return nil, fmt.Errorf("marshalling acs client request data: %w", err)
+	}
+	req, err := http.NewRequest("POST", fmt.Sprintf("%s/apis/beta/acs/v1", c.realmConfig.BaseURL), bytes.NewBuffer(bodyBytes))
+	if err != nil {
+		return nil, fmt.Errorf("creating acs client request: %w", err)
+	}
+	req.Header.Add("Content-Type", "application/json")
+	req.Header.Add("Authorization", fmt.Sprintf("Bearer %s", accessToken))
+
+	resp, err := client.Do(req)
+	if err != nil {
+		return nil, fmt.Errorf("sending acs client request: %w", err)
+	}
+
+	if !httputil.Is2xxStatusCode(resp.StatusCode) {
+		return nil, fmt.Errorf("error creating acs client [%d]", resp.StatusCode)
+	}
+
+	defer shared.CloseResponseBody(resp)
+	responseBody, err := ioutil.ReadAll(resp.Body)
```

Comment:
ioutils has been deprecated in favor of io / os package:
```suggestion
	responseBody, err := io.ReadAll(resp.Body)
```